.github/workflows/_test-tf.yaml

```diff
@@ -3,9 +3,9 @@ on:
     branches:
       - main
     paths:
-      - '.github/workflows/_test-tf.yaml'
-      - '.github/workflows/tf-*.yaml'
-      - 'tests/terraform/**'
+      - ".github/workflows/_test-tf.yaml"
+      - ".github/workflows/tf-*.yaml"
+      - "tests/terraform/**"
```

Comment:
Im not a fan of changing quotes like this as it doesnt bring anything and just adds noise